scalardb-sample/README.md

```diff
@@ -1,162 +1,125 @@
-# ScalarDB Sample
+# Create a Sample Application by Using ScalarDB
```

Comment:
I'd like to change this file name from `README.md` to `index.md` so that the link on the docs site looks a bit more standard. For example:

- Current file name (`README.md`) URL: https://scalardb.scalar-labs.com/docs/latest/scalardb-samples/scalardb-sample/README/
- Proposed file name (`index.md`) URL: https://scalardb.scalar-labs.com/docs/latest/scalardb-samples/scalardb-sample/

Any concerns about this change? If not, I'd like to apply the same change to the other samples in this repository.

Reply:
Do you know if `index.md` is rendered as well as `README.md` on GitHub when opening each sample application directory (e.g., https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) ? That's my slight concern.

Reply:
@komamitsu I see what you mean. Unfortunately, no. Renaming `README.md` to `index.md` will not make the contents appear as it does now when the user visits the folder that contains the sample. (Instead, the user would have to open the `index.md` to view its contents.)

With that in mind, I'll leave the file name as is for (`README.md`) now since it may be something I specify on the docs site side instead.